tests/pom.xml

```diff
@@ -245,6 +240,42 @@
   </dependencies>
 
   <profiles>
+
+    <!-- Profile to build tests module connecting to hdfs 1 -->
+    <profile>
+      <id>hadoop-1</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-core</artifactId>
```

Comment:
for hadoop-1, `hadoop-client` hides a few deps from `hadoop-core`, leading missing class error in test. So we need to have the direct dep on `hadoop-core` rather than `hadoop-client` which is a wrapper on `hadoop-core`